csrc/compute_at_map.cpp

```diff
@@ -299,26 +299,26 @@ findFirstSelfMapping(Fusion* fusion, const IterDomainGraph& id_graph) {
     // and it maybe enough to check the root domains.
 
     // Root domains
-    auto self_mappped_root_pair =
-        detectMappablePair(tv->getRootDomain(), id_graph, IdMappingMode::EXACT);
+    auto self_mappped_root_pair = detectMappablePair(
```

Comment:
nit: the comment above should be rfactor domains

Reply:
updated